app/models/foreman_ansible/ansible_provider.rb

```diff
@@ -20,9 +20,16 @@ def proxy_command_options(template_invocation, host)
           super(template_invocation, host).merge(
             'ansible_inventory' => ::ForemanAnsible::InventoryCreator.new(
               [host], template_invocation
-            ).to_hash.to_json
+            ).to_hash.to_json,
+            :remote_execution_command => ansible_command?(template_invocation.template)
           )
         end
+
+        private
+
+        def ansible_command?(template)
+          template.job_category == 'Ansible Commands'
```

Comment:
Could we use the rex feature instead of a category? This would still generate reports for e.g. service restart or package install. We only want that for roles plays. So it would be something like unless feature is play roles.